airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml

```diff
@@ -32,7 +32,7 @@ acceptance_tests:
         skip_comprehensive_incremental_tests: true
         configured_catalog_path: integration_tests/configured_catalog.json
         future_state:
-          future_state_path: integration_tests/abnormal_state.json
+          bypass_reason: "Concurrent incremental streams emit a state with the most recent record or the lower boundary instead of the incoming abnormal state value"
```

Comment:
See slack convo for more context: https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1730329381148769

basically now that we rely on the lower boundary of a date interval when checkpointing, we can no longer run the assertion on the abnormally large future state. This test is of dubious value so we may want to explore just tearing out the whole test. However, I don't want this to block the release of this connector